jet/dashboard/dashboard_modules/google_analytics.py

```diff
@@ -16,14 +16,14 @@
 import httplib2
 from jet.dashboard.modules import DashboardModule
 from oauth2client.client import flow_from_clientsecrets, OAuth2Credentials, AccessTokenRefreshError, Storage
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
 from django.conf import settings
-from django.utils.encoding import force_text
+from django.utils.encoding import force_str
```

Comment:
`smart_text() and force_text()` are deprecated in favor of `smart_str() and force_str()`.